lib/generateMarkdown.js

```diff
@@ -73,7 +73,7 @@ function generateMarkdown(options) {
         }
 
         // Render section for each property
-        md += createPropertiesDetails(schema, title, options.headerLevel + 1, options.suppressWarnings);
+        md += createPropertiesDetails(schema, '', options.headerLevel + 1, options.suppressWarnings);
```

Comment:
I wasn't sure how you wanted to handle titles since I added recursion.  In the old method, if I had a title that was (in my case) "CI Pipeline Environment Definition", then I ended up with

# CI Pipeline Environment Definiton.override_attributes

#### CI Pipeline Environment Definition.override_attributes.weeverapps.docker

And when things got nested the heading was extremely long.  I considered using JSONPath notation, but I just figured the root (`$` in JSONPath) was implied.  Also, since the title is displayed at the top of the document, I found it redundant to show it in every heading as well.

Open to suggestions on this, but my gut is leaning towards leaving it as-is - starting with a `.` - or using JSONPath like:

# $.override_attributes